### PR TITLE
MOSIP-22709

### PR DESCRIPTION
**Pull request by NagarjunaKuchi**

### CODE REVIEW

File: db_release_scripts/mosip_pms/sql/1.2.1_pms-scripts_release.sql [hunk 1]
```diff
@@ -20,3 +20,20 @@ ALTER TABLE pms.partner_h ADD COLUMN addl_info character varying;
 
 INSERT INTO pms.partner_type (code,partner_description,is_policy_required,is_active,cr_by,cr_dtimes,upd_by,upd_dtimes,is_deleted,del_dtimes) VALUES
 	 ('Print_Partner','Print Partner',true,true,'superadmin',now(),NULL,NULL,false,NULL);
+
+UPDATE pms.partner_type SET is_active=true, upd_dtimes =now(), upd_by ='superadmin' WHERE code='MISP_Partner';
```
Comment (gsasikumar) [inline db_release_scripts/mosip_pms/sql/1.2.1_pms-scripts_release.sql:24]: we want to activate all?

File: db_release_scripts/mosip_pms/sql/1.2.1_pms-scripts_revoke.sql [hunk 2]
```diff
@@ -16,4 +16,8 @@ ALTER TABLE pms.partner_h DROP COLUMN logo_url;
 
 ALTER TABLE pms.partner DROP COLUMN addl_info;
 
-ALTER TABLE pms.partner_h DROP COLUMN addl_info;
\ No newline at end of file
+ALTER TABLE pms.partner_h DROP COLUMN addl_info;
+
+UPDATE pms.partner_type SET is_active=false,upd_dtimes =now(), upd_by ='superadmin' WHERE code ='MISP_Partner'
```
Comment (gsasikumar) [inline db_release_scripts/mosip_pms/sql/1.2.1_pms-scripts_revoke.sql:21]: is it ok to activate all? What if someone has deactivated a partner earlier?

**Review by gsasikumar — CHANGES_REQUESTED** [review]

File: db_release_scripts/mosip_pms/sql/1.2.1_pms-scripts_release.sql (hunk 1 above)
Comment (ase-101) [reply db_release_scripts/mosip_pms/sql/1.2.1_pms-scripts_release.sql:24]: it is not is_active, it should be is_policy_required set to true.

File: db_release_scripts/mosip_pms/sql/1.2.1_pms-scripts_revoke.sql (hunk 2 above)
Comment (ase-101) [reply db_release_scripts/mosip_pms/sql/1.2.1_pms-scripts_revoke.sql:21]: it is not is_active, it should be is_policy_required set to false in this revoke script.

Comment (ase-101) [reply db_release_scripts/mosip_pms/sql/1.2.1_pms-scripts_revoke.sql:21]: We will be raising a new PR with the change sasi.

**Pull request merged**